platform_versioned_docs/version-24.1/limits/limits.mdx

```diff
@@ -9,37 +9,37 @@ Seqera Platform elements and features have default limits per workspace and orga
 
 ### Workspaces
 
-| Description  | Value |
-| ------------ | ----- |
-| Active runs  | 5     |
-| Members      | 50    |
-| Participants | 50    |
-| Pipelines    | 100   |
-| Datasets     | 100   |
-| Labels       | 1000  |
+| Description  | Free | Cloud Pro + Enterprise |
+| ------------ | ---- | ---------------------- |
+| Active runs  | 5    | 100, or per license    |
+| Members      | 50   | 50, or per license     |
+| Participants | 50   | 50, or per license     |
+| Pipelines    | 100  | 100, or per license    |
+| Datasets     | 100  | 1000, or per license   |
+| Labels       | 1000 | 1000, or per license   |
```

Comment:
For enterprise, the code sets the default max amount of labels per workspace to 100 unless a limit is set in the license.
By default, new licenses are created with a 1000 limit for labels. Is that the default you are referring to?

Reply:
Yes, referring to the default license limit here. There's also a note on the page that mentions that older licenses may have a limit of 100 and people should contact support for an updated license if needed.